tests/test_hub.py

```diff
@@ -152,17 +152,17 @@ def test_push_metric_invalid_dataset_type(self, known_task_ids, metadata_update)
                 task_type="dummy-task",
             )
 
-            assert "Dataset dataset_type not found on hf.co/datasets" in self._caplog.text
+            assert "Dataset dataset_type not found on the Hub at hf.co/datasets/dataset_type" in self._caplog.text
```

Comment:
shouldn't this be 
```suggestion
            assert f"Dataset dataset_type not found on the Hub at hf.co/datasets/{dataset_type}" in self._caplog.text
```

Reply:
Nope, `dataset_type` isn't a variable in that scope so I think that would fail. "dataset_type" is the I tend to not use templated strings for asserts unless the test is parametrized.